File: scratch/read_configs.js
const sqlite3 = require('sqlite3').verbose();
const path = require('path');

const dbPath = path.join(__dirname, '..', 'backend', 'vagasync.db');
const db = new sqlite3.Database(dbPath, (err) => {
  if (err) {
    console.error('Error opening database', err);
    return;
  }
  console.log('Database opened successfully');
});

db.all("SELECT * FROM configs", [], (err, rows) => {
  if (err) {
    console.error('Error querying configs', err);
    return;
  }
  console.log('--- Configs ---');
  rows.forEach((row) => {
    if (row.key === 'resume_text') {
      console.log(`key: ${row.key}, length: ${row.value ? row.value.length : 0}`);
      console.log(`preview: ${row.value ? row.value.substring(0, 100) : 'null'}`);
    } else {
      console.log(`key: ${row.key}, value: ${row.value}`);
    }
  });
  db.close();
});

File: scratch/read_configs.py
import sqlite3
import os

db_path = os.path.join(os.path.dirname(__file__), '..', 'backend', 'vagasync.db')
conn = sqlite3.connect(db_path)
cursor = conn.cursor()

cursor.execute("SELECT key, value FROM configs")
rows = cursor.fetchall()

print("--- Configs ---")
for key, value in rows:
    if key == 'resume_text':
        print(f"key: {key}, length: {len(value) if value else 0}")
        print(f"preview: {value[:100] if value else 'None'}")
    else:
        print(f"key: {key}, value: {value}")

conn.close()
